numba_dpex/examples/kernel/vector_sum2D.py

```diff
@@ -22,7 +22,7 @@ def data_parallel_sum(a, b, c):
 
 
 def driver(a, b, c, global_size):
-    data_parallel_sum[global_size](a, b, c)
+    data_parallel_sum[global_size, dpex.DEFAULT_LOCAL_SIZE](a, b, c)
```

Comment:
Ok. But, how about demonstrating the use of the new and likely future method of using Range classes?

Reply:
A new driver function is added to showcase the new way of specifying kernel lauch params.